tests/integ/test_chainer_train.py

```diff
@@ -35,6 +35,8 @@ def test_distributed_cpu_training(sagemaker_session, chainer_full_version):
     _run_mnist_training_job(sagemaker_session, "ml.c4.xlarge", 2, chainer_full_version)
 
 
+@pytest.mark.skipif(REGION in ['us-west-1', 'eu-west-2', 'ca-central-1'],
+                    reason='No ml.p2.xlarge supported in these regions')
```

Comment:
what about p3? can't we use p3 then?

Reply:
P3 is either not supported or supported only by 1/3 or availability zones. So for these 3 regions we just skip tests for now to make canary good for now.